dipy/workflows/tests/test_reconst_dki.py

```diff
@@ -0,0 +1,93 @@
+from os.path import join
+
+import nibabel as nib
+from nibabel.tmpdirs import TemporaryDirectory
+
+import numpy as np
+
+from nose.tools import assert_true
+
+from dipy.data import get_data
+from dipy.workflows.reconst import ReconstDkiFlow
+
+
+def test_reconst_dki():
+    with TemporaryDirectory() as out_dir:
+        data_path, bval_path, bvec_path = get_data('small_101D')
+        vol_img = nib.load(data_path)
+        volume = vol_img.get_data()
+        mask = np.ones_like(volume[:, :, :, 0])
+        mask_img = nib.Nifti1Image(mask.astype(np.uint8), vol_img.affine)
+        mask_path = join(out_dir, 'tmp_mask.nii.gz')
+        nib.save(mask_img, mask_path)
+
+        dki_flow = ReconstDkiFlow()
+
+        args = [data_path, bval_path, bvec_path, mask_path]
+
+        dki_flow.run(*args, out_dir=out_dir)
+
+        fa_path = dki_flow.last_generated_outputs['out_fa']
+        fa_data = nib.load(fa_path).get_data()
+        assert_true(fa_data.shape == volume.shape[:-1])
+
+        tensor_path = dki_flow.last_generated_outputs['out_dt_tensor']
+        tensor_data = nib.load(tensor_path)
+        assert_true(tensor_data.shape[-1] == 6)
```

Comment:
I think you prefer  `assert_equal` than `assert_true` like you said on #1387 ;-)